### PR TITLE
Adding swagger spec for unprepare subnet

### PR DESCRIPTION
**Pull request by mvvsubbu**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by number213** [comment]
Spellchecker is failing because it doesn't recognize the word "unprepare" and it's variants. My suggestion is to add them to `custom-words.txt` that 's located at repo's root

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json [hunk 1]
```diff
@@ -593,6 +593,67 @@
...
+          },
+          {
+            "name": "subnetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the subnet."
+          },
+          {
+            "name": "unprepareNetworkPoliciesRequestParameters",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: Most specs in Network use `parameters` as a name for body object. Please consider renaming this for consistency.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json [hunk 2]
```diff
@@ -593,6 +593,67 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/subnets/{subnetName}/UnprepareNetworkPolicies": {
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: Minor nitpick, but if URL is case-insensitive, please consider using camelCase for `UnprepareNetworkPolicies` for the sake of consistency

**Review by anton-evseev — APPROVED** [review]
Looks good aside for a couple nitpicks for consistency's sake

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json (hunk 1 above)
Comment (mvvsubbu) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json]: It is body object only.

**Comment by mvvsubbu** [comment]
@MikhailTryakhov and @Am018 can you review this

**Comment by number213** [comment]
@Azure/arm-api-review-board could you please review this PR?

**Comment by number213** [comment]
@ryansbenson could you please review this PR?

**Comment by number213** [comment]
Retargeted to `master` since `network-july-release` got merged

**Comment by ravbhatnagar** [comment]
Signing off from ARM side. One small comment - can the service name be an enum? if not, may be update the description to provide a few examples.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**